opentera-webrtc-native-client/OpenteraWebrtcNativeClient/python/CMakeLists.txt

```diff
@@ -2,30 +2,29 @@ cmake_minimum_required(VERSION 2.6.4)
 
 project(opentera_webrtc_native_client)
 
-set(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR}/bin/${CMAKE_BUILD_TYPE})
-
```

Comment:
Unknown usage

Reply:
Easy to find bin directory directly under build/bin for launching tests.  Why not leaving that way ?

Reply:
I'm pretty sure `${CMAKE_CURRENT_BINARY_DIR}` is already a path that contains `${CMAKE_BUILD_TYPE}`, which means it is redundant.
We could remove that part, then we would basically move the built .so file back one level from `${CMAKE_CURRENT_BINARY_DIR}` to `${CMAKE_CURRENT_BINARY_DIR}/bin`. Is that what we want?